.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected
+
+---
+
+# Bug Report
+
+## Description
+<!-- A clear and concise description of what the bug is. -->
+<!-- Providing a link to a live example / minimal demo of the problem greatly helps us debug issues. -->
+
+## Steps to Reproduce
+<!-- Please specify the exact steps you took for this bug to occur. -->
+<!-- Provide as much detail as possible so we're able to reproduce these steps. -->
+1.
+2.
+3.
+4.
+
+## Additional Context / Screenshots
+<!-- Add any other context about the problem here. If applicable, add screenshots to help explain. -->
+
+## Expected Behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Possible Solution
+<!--- If you have suggestions to fix the bug, let us know -->
+
+## Environment
```

Comment:
Not sure if we need all of these sections for this repo

Reply:
Lemme know what y'all want these to look like. Just copy/pasted from another terra repo.

Reply:
Let's delete the whole `Environment` section and keep the rest of the file as is